dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -127,6 +127,15 @@ def toggle_hidden
     render json: []
   end
 
+  def instructions
+    require_levelbuilder_mode
+    authorize! :read, ScriptLevel
+
+    script = Script.get_from_cache(params[:script_id])
+
+    render 'levels/instructions', locals: { stages: script.stages }
```

Comment:
why? i feel like local_assigns makes it much more explicit what data I'm passing from the controller to the view.